Packs/ThreatGrid/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Cisco Secure Malware Analytics",
     "description": "Query and upload samples to Cisco threat grid.",
     "support": "xsoar",
-    "currentVersion": "2.0.22",
+    "currentVersion": "2.1.0",
```

```suggestion
    "currentVersion": "2.0.23",
```
I think this should be a minor change.